include/exadg/utilities/print_general_infos.h

```diff
@@ -24,42 +24,22 @@
 
 // deal.II
 #include <deal.II/base/conditional_ostream.h>
-#include <deal.II/base/revision.h>
 #include <deal.II/base/utilities.h>
 #include <deal.II/distributed/tria_base.h>
+#include <deal.II/fe/mapping_q.h>
```

Comment:
Why including mapping here? Is it really needed by this file or somewhere else?

Reply:
It is needed in ```print_grid_info()``` and after splitting the files, there were errors that MappingQ is not known

Reply:
I see, we should in the future shift the other implementations also to the cpp file.